app/assets/images/payment_icons/dpo-airtelmoney.svg

```diff
@@ -0,0 +1,16 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="38" height="24" viewBox="0 0 38 24" role="img"
```

Comment:
The border radius doesn't match other payment icons.

Reply:
Icons have been updated